src/Microsoft.Azure.WebJobs.Logging/Entities/InstanceTableEntity.cs

```diff
@@ -90,9 +89,14 @@ internal static string RowKeyFunctionInstanceId(Guid functionInstanceId)
 
         public string TriggerReason { get; set; }
 
-        public DateTimeOffset StartTime { get; set; }
+        public DateTime StartTime { get; set; }
 
-        public DateTimeOffset? EndTime { get; set; }
+        public DateTime? EndTime { get; set; }
+
+        // Last heart beat. When EndTime is missing, this lets us guess if the function is still alive. 
+        // Whereas Start and End are written very determinsitically in the lifecycle, 
+        // heartbeat is special because it's written at potentially random times by a background poller thread. 
```

Comment:
Where does this poller thread actually live? Where are these updates being emitted?